build/pr-pipeline.yml

```diff
@@ -83,20 +83,22 @@ stages:
           catch
           {}
           New-AzResourceGroup -Name "$(ResourceGroupName)" -Location "$(ResourceGroupRegion)" -Force
-
-- stage: aadTestEnvironment
-  displayName: Setup AAD Test Environment
-  dependsOn:
-  - provisionEnvironment
-  - DockerBuild
-  jobs:
-  - template: ./jobs/add-aad-test-environment.yml
+  - template: ./jobs/cleanup-aad.yml
```

Comment:
The provision environment stage now needs to depend on the SemVer stage since add AAD uses the environment name.

Reply:
Thanks! I think I've keep the provision stage separate and move the aad cleanup/create to its own stage

Reply:
Dependencies now look like this. 
![image](https://user-images.githubusercontent.com/197221/110002947-c5e09580-7cca-11eb-801c-230ec235b4a4.png)